exercises/diffie-hellman/README.md

```diff
@@ -37,10 +37,6 @@ Bob calculates
 The calculations produce the same result!  Alice and Bob now share
 secret s.
 
-## Notes
```

Comment:
This README (and the one for `simple-cipher`) will need either regenerating to include the hint. Alternatively, you can just copy-and-paste it in at this location.

Reply:
Ready

Reply:
@lucasloisp, the section needs to go before `### Submitting Exercises`, since that's where it goes if we regenerate the file.